imports/plugins/core/graphql/server/resolvers/account/Query/roles.js

```diff
@@ -0,0 +1,25 @@
+import { getPaginatedRolesResponse } from "@reactioncommerce/reaction-graphql-xforms/roles";
+import { decodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+
+/**
+ * @name roles
+ * @method
+ * @summary find and return the roles for a shop
+ * @param {Object} _ - unused
+ * @param {Object} args - an object of all arguments that were sent by the client
+ * @param {String} args.shopId - id of shop to query
+ * @param {String} args.after - Connection argument
+ * @param {String} args.before - Connection argument
+ * @param {Number} args.first - Connection argument
+ * @param {Number} args.last - Connection argument
```

Comment:
Mike made a custom type you can reference in the jsdocs here instead of every pagination arg which are the same everywhere